packages/core/src/components/graphs/confidence-interval.ts

```diff
@@ -0,0 +1,208 @@
+// Internal Imports
+import { Component } from "../component";
+import * as Configuration from "../../configuration";
+import {
+	CartesianOrientations,
+	Events,
+	ColorClassNameTypes
+} from "../../interfaces";
+import { DOMUtils } from "../../services";
+import { Tools } from "../../tools";
+import settings from "carbon-components/es/globals/js/settings";
+
+// D3 Imports
+import { area } from "d3-shape";
+import { select } from "d3-selection";
+
+export class ConfidenceInterval extends Component {
+	type = "area";
+
+	init() {
+		const eventsFragment = this.services.events;
+
+		// Highlight correct area on legend item hovers
+		eventsFragment.addEventListener(
+			Events.Legend.ITEM_HOVER,
+			this.handleLegendOnHover
+		);
+
+		// Un-highlight area on legend item mouseouts
+		eventsFragment.addEventListener(
+			Events.Legend.ITEM_MOUSEOUT,
+			this.handleLegendMouseOut
+		);
+	}
```

Comment:
we probably don't need these since we're only visualizing 1 datagroup

Reply:
Yep makes sense and has been removed!